src/styles/blocks/body.css

```diff
@@ -0,0 +1,3 @@
+.body {
+  margin: 0;
+}
```

Comment:
Looks like you want to reset the default browser margin, for this purpose file reset.css suits better.